api/server/boot/routes.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(server) {
```

Comment:
This file is automatically created.
as mentioned in https://docs.strongloop.com/display/public/LB/Logging+in+users